presidio-analyzer/analyzer/analyzer_engine.py

```diff
@@ -108,6 +118,10 @@ def analyze(self, text, entities, language, all_fields):
         # run the nlp pipeline over the given text, store the results in
         # a NlpArtifacts instance
         nlp_artifacts = self.nlp_engine.process_text(text, language)
+
+        self.app_tracer.trace(request_id, "nlp artifacts:"
+                              + repr(nlp_artifacts))
```

Comment:
`repr` is usually used for debugging and is usually a short version of the object's values. If we want to store the entire object into the log, we should provide a method for nlp_artifacts to serialize itself, such as `to_dict` or `to_json`.